final draft with logo image

ld051609 opened the pull request:
We can't wait to see your new miniapp live on seam.so and the Seam mobile apps! Fill out the below sections to get ready for review.

# [RiddleQuest]
### [Description of miniapp]
- Seam username: laniD
- Provides users with daily riddles, allowing them to interact by submitting their answers and rating the difficulty of each riddle. Then, the post is shared publicly within the seam

nconfrey commented:
Thanks for your submission to the Seam Miniapp Challenge! For better review, please add photos / videos of the miniapp in action.

Feel free to update this pull request until the deadline of 5pm EST today. We'll start reviewing these soon and might have some code review comments to make your submission even better!

nconfrey reviewed (changes requested):
Hi @ld051609 ! I'm not able to find the Seam user `laniD` -- you'll need to setup a Seam account in order to finalize the submission of the miniapp!

ld051609 commented:
Hi,
I just figured out that I created an account under the different name,
however, I deleted that account and changed into a new name "lanid" - i
don't know why i cannot capitalize the word "D", so "lanid" instead of
"laniD"
Thanks for letting me know.


On Mon, Aug 19, 2024 at 4:17 PM nconfrey ***@***.***> wrote:

> ***@***.**** requested changes on this pull request.
>
> Hi @ld051609 <https://github.com/ld051609> ! I'm not able to find the
> Seam user laniD -- you'll need to setup a Seam account in order to
> finalize the submission of the miniapp!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/seam-xyz/Miniapp-Builder/pull/96#pullrequestreview-2246382395>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BER3LFU7OTZRF7JWAI65LD3ZSJHEFAVCNFSM6AAAAABMWWMMEOVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDENBWGM4DEMZZGU>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

nconfrey reviewed (approved):
Awesome, nice work @ld051609 ! Congratulations, your miniapp has been chosen as a finalist for the miniapp challenge, and we'll be merging it in shortly. Expect to see it live in the Seam iOS app and website (at seam.so) by the end of this week!!
